### PR TITLE
fix: hyperchain setup

### PR DESCRIPTION
**Pull request by lgalabru**
Couple of adjustments on the hyperchain integration, along with the ability to spin up a dedicated api container.
Todo:
- [x] Use latest hyperchain contract - to be deployed by @pavitthrap

### CODE REVIEW

File: src/generate/project.rs [hunk 1]
```diff
@@ -376,7 +376,7 @@ disable_stacks_api = false
 # bitcoin_explorer_image_url = "quay.io/hirosystems/bitcoin-explorer:devnet"
 # postgres_image_url = "postgres:alpine"
 # enable_hyperchain_node = true
-# hyperchain_image_url = "hirosystems/hyperchains:103-merge-stretch"
+# hyperchain_node_image_url = "hirosystems/hyperchains:103-merge-stretch"
```
Comment (obycode) [inline src/generate/project.rs:379]: Should this match the default? (`hirosystems/hyperchains:0.0.2-stretch`)

**Review by obycode — APPROVED** [review]
Looks good! Just one minor comment.

File: src/generate/project.rs (hunk 1 above)
Comment (lgalabru) [reply src/generate/project.rs]: Addressed with latest

**Review by obycode — APPROVED** [review]
🚀

**Pull request merged**